docs/reference/style/pack.rst

```diff
@@ -465,3 +458,104 @@ specifying the allocated width and allocated height.
       If the parent element has an ``alignment`` value of ``center``, the
       horizontal position of the child is set to 1/2 of the extra width,
       relative to the parent.
+
+4. **Evaluate final width and height**
+
+   If the element has no children, the final width of the element is set to
+   the available width, and the final height of the element is set to the
+   available height.
+
+   Otherwise, the computed width and height of the element is set to width and
+   height of the box that contains all the children and their padding.
+
+   If the element is the root element of a layout, and the final width and height
+   are less than the available view width and height, the final width and height
+   are expanded to match the available view width and height.
+
+The relationship between Pack and CSS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pack aims to be a functional subset of CSS. Any Pack layout can be converted
+into an equivalent CSS layout. After applying this conversion, the CSS layout
+should be considered a "reference implementation". Any disagreement between the
+rendering of a converted Pack layout in a browser, and the layout produced by
+the Toga implementation of Pack should be considered to be either a bug in Toga,
+or a bug in the mapping.
+
+The mapping that can be used to establish the reference implementation is:
+
+* The reference HTML layout document is rendered in `standards mode
+  <https://developer.mozilla.org/en-US/docs/Web/API/Document/compatMode>`__,
+  with a default CSS stylesheet:
+
+  .. code-block:: html
+
+      <!DOCTYPE html>
+      <html>
+         <head>
+           <meta charset="UTF-8" />
+            <title>Pack layout testbed</title>
+            <style>
+               html, body {
+                  height: 100%;
+                  white-space: nowrap;
+               }
+               body {
+                  overflow: hidden;
+                  display: flex;
+                  margin: 0;
+               }
+               div {
+                  display: inline-flex;
+                  margin: 0;
```

Comment:
```suggestion
               div {
                  display: flex;
```
According to [this](https://www.w3.org/TR/css-flexbox-1/#flex-items), "The display value of a flex item is blockified", so there's no difference between `flex` and `inline-flex` in this context.

There's also no need for a `margin`, because unlike `body`, no browser will have a default margin on `div`.

Reply:
Agreed on `margin`; I swear there was a reason I used inline-flex, but that appears to be lost to the depths of time. I can't find any case where it matters.